### PR TITLE
Address Power Platform pilot comments and substantive changes in the baseline document

### LINKED ISSUE #239: Adjudicate PowerPlatform comments on the comment matrix
- [ ] Send out link to Comments Matrix to the Baseline POCs
- [ ] Andrew to schedule meetings with Baseline POCs
- [ ] Internally review (Andrew & Baseline POCs) PowerPlatform comments from the comments matrix 
- [ ] Review comments with CISA
- [ ] Open GitHub Issues as needed
- [ ] Prioritize GitHub Issues into release lifecycle

### LINKED ISSUE #287: Adjudicate "substantive changes" to the PowerPlatform Baseline Markdown
# 💡 Summary #

This issue captures the high-level tasks required to convert the existing PowerPlatform Secure Configuration Baseline document to a new structure based on the existing markdown format.  The updates will include reordering and regrouping of existing policy items, additional fields for each policy item, and policy changes to clarify relative to existing implementation guidance.

## Motivation and context ##

The new baseline policy document format will provide additional clarity to readers and support better automated assessment of baseline policy items for PowerPlatform.

## Implementation notes ##

In order to make the changes referenced above, the following tasks will need to be accomplished:

- [ ] CREATE and NUMBER policy group statements based on existing policy areas using updated markdown files found in #212 and its associated branch 
   - [ ] REMOVE “SHALL” and “SHOULD” language from new policy group statements
   - [ ] RENAME policy groups to cover associated policies (Naming should be descriptive vice prescriptive)
   - [ ] REGROUP baseline policies that are better orientated to the new Group Name if needed
   - [ ] CREATE short 1-2 sentence summary for each policy group

- [ ] ADD `Rationale` section and text for each policy
    _Note: Suggested format for Rationale_
    - 1st sentence: Security risk if policy is not implemented.
    - 2nd sentence: How this policy minimizes this risk.

- [ ] UPDATE policy item description to match the desired best practice and synchronize with implementation steps (e.g., setting should be 30 days or less when implementation shows setting days to <= 30)

## Acceptance criteria ##

How do we know when this work is done?

- [ ] PowerPlatform baseline markdown has been updated with new policy groups and all policy items reordered into those groups
- [ ] All PowerPlatform baseline policy groups have been numbered
- [ ] Each PowerPlatform policy group contains a concise and clear 1-2 sentence summary
- [ ] Each PowerPlatform policy item contains a `Rationale` section using the suggested format
- [ ] Each PowerPlatform policy item has been reviewed and updated to synchronize with the implementation steps

### LINKED ISSUE #311: Recommended new Power Platform baseline policy - Restrict the creation of Power Pages to administrators only.
# 💡 Summary #
Microsoft in their review of Power Platform recommended adding a new Power Platform baseline policy that restricts the creation of [Power Pages (websites) to admin users](https://learn.microsoft.com/en-us/power-apps/maker/portals/control-portal-creation) 
This will restrict the creation of the Apps to users to the following roles.
- Global Admin,
- Power Platform Admin
- Dynamic 365 Admin

This control currently can only be implemented via PowerShell i.e no GUI. This setting can be checked via ScubaGear as the cmdlet that retrieves this setting is already invoked in the Power Platform Provider.
```
Set-TenantSettings -RequestBody @{ "disablePortalsCreationByNonAdminUsers" = $false }
```
## Question for CISA ##
- Should we add this new policy to the Power Platform baseline?
- Are there use cases where non-administrative users would want to create Power Pages?

### PR DESCRIPTION
**Pull request by buidav**
## 🗣 Description ##

- Reformatted the baseline document with new structure and policy identifiers
- Added additional language and resources in the intro to address pilot feedback
- Added a new Policy Group for Power Pages
- Refine and update policy statements to address pilot feedback. 
- Updated implementation steps for any out of date instructions

TODO: address code updates in #399

## 💭 Motivation and context ##
closes #239 
closes #287
closes #311 

## 🧪 Testing ##
- Retested instructions for the Power Platform policies on commercial and GCC tenants.
- Addressed comments in the comments matrix.

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - *eschew scope creep!*
- [ ] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Review by rgbrow1949 — CHANGES_REQUESTED** [review]
Found grammatical errors. I'm still reviewing.

**Review by rgbrow1949 — CHANGES_REQUESTED** [review]
Grammar Notes: 
- (Line 27) drop "that".
- (Line 29) add comma after "get notifications".
- (Lines 35-37) Put a comma after "a Power App" before the "or". Also, make the objects of the sentence: "proxy" and wrapper", plural. So: "These are proxies or wrappers around an API that allow the underlying service to be accessed from Power Automate Workflows, a Power App, or Azure Logic Apps. 
- (Line 42) Comma after "update".
- (Line 80) Comma after "Power Apps".
- (Line 92) "controls" should be singular: 'control'.
- (Line 98) Drop "the" before "Global Admin".
- (Line 119, 236, 325, 371) Is it supposed to read: "your tenant's respective Power Platform Admin Center"? If so, why is "Power Platform admin Center" in square brackets? 
- (Line 137) Put a comma after "To secure Power Platform environments".
- (Line 150-151) Drop the comma after "blocked".
- (Line 154) Should it be "in the Power Platform" or just "in Power Platform"?
- (Line 217) Agency is mispelled.
- (Line 256-278, 334-337, 371-384) Inconsistent spaces before instruction # compared to previous and future amount of spaces before instruction numbers. (1 space vs. 2 spaces).
- (Line 254) Category should not be capitalized. Should be "category".
- (Line 349) Comma after clickjacking.

**Review by rgbrow1949 — APPROVED** [review]

File: baselines/powerplatform.md [hunk 1]
```diff
@@ -4,42 +4,40 @@ The Microsoft Power Platform is a group of applications involving
 low-code application development, business intelligence, a custom chat
 bot creator, and app connectivity software. The following summarizes the
 Power Platform applications and other applications frequently used by
-Power Platform applications.
+Power Platform.
```
Comment (nanda-katikaneni) [inline baselines/powerplatform.md:8]: Please update the title as per new style guide. (viz. CISA M365 Security Configuration Baseline for Power Platform)

File: baselines/powerplatform.md [hunk 2]
```diff
@@ -222,45 +230,52 @@ All connectors except those listed below SHOULD be added to the Blocked category
...
 
-14. Select **Next**-\> **Create Policy** to finish.
+15.  Select **Next**-\> **Create Policy** to finish.
 
-## 3. Tenant Isolation
+#### MS.POWERPLATFORM.2.2v1 instructions:
+1.  Repeat the steps above but for step **13** select the non-default environment you wish to enforce a DLP policy upon.
+
+#### MS.POWERPLATFORM.2.3v1 instructions:
+1.  Refer to steps **8** and **9** in the **MS.POWERPLATFORM.2.3v1** instructions to meet this policy.
```
Comment (nanda-katikaneni) [inline baselines/powerplatform.md:276]: This step within MS.POWERPLATFORM.2.3v1 refers to itself for nonexistent steps 8&9 - a typo? seems reference should be to "2.1v1" steps 8&9.

**Review by nanda-katikaneni — CHANGES_REQUESTED** [review]
A few styling and incorrect reference to be fixed.

**Pull request closed without merging**